equinix/fabric_port_mapping_helper.go

```diff
@@ -104,26 +104,26 @@ func encapsulationToTerra(portEncapsulation *v4.PortEncapsulation) *schema.Set {
 	return portEncapsulationSet
 }
 
-func lagToTerra(portLag *v4.PortLag) *schema.Set {
-	if portLag == nil {
-		return nil
-	}
-	portLags := []*v4.PortLag{portLag}
-	mappedPortLags := make([]interface{}, 0)
-	for _, portLag := range portLags {
-		mappedPortLag := make(map[string]interface{})
-		mappedPortLag["enabled"] = portLag.Enabled
-		mappedPortLag["id"] = portLag.Id
-		mappedPortLag["name"] = portLag.Name
-		mappedPortLag["member_status"] = portLag.MemberStatus
-		mappedPortLags = append(mappedPortLags, mappedPortLag)
-	}
-	portLagSet := schema.NewSet(
-		schema.HashResource(readPortLagRes),
-		mappedPortLags,
-	)
-	return portLagSet
-}
+//func lagToTerra(portLag *v4.PortLag) *schema.Set {
```

Comment:
Deleting code is preferable to commenting it out; if we needed to bring it back later we can always find it in the git history.

Reply:
Ack!